bin/.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+/** @format */
+
+module.exports = {
+	extends: '../.eslintrc.js',
+	rules: {
+		'import/no-extraneous-dependencies': 0,
```

Comment:
If I remember right, this one fails when you try to require something that's not listed in deps or devdeps. We've gotten into trouble in the past require'ing things that were only transitive dependencies. 

I turned the rule back on and there's only one violation:
```
wp-calypso/bin/build-metadata.js
    2:17  warning  'request' should be listed in the project's dependencies. Run 'npm i -S request' to add it  import/no-extraneous-dependencies
```

Reply:
Yep, makes sense to drop that one.

Done in https://github.com/Automattic/wp-calypso/pull/28167/commits/cb2ed65ab873a643cab923949c5e18bc43bfbb04